oscarnevarezleal/prr
# chores

## What?
- Added a feature to allow `prr` to read the last commits and include them in the PR description. 
- Updated the README to include usage and requirements information. 
- Included person and tone configurations and simplified the command structure. Added initial commands including `init` and `generate`.

## How?
- Updated the `prr` functionality to read the last commits.
- Updated the README with comprehensive usage and requirements details.
- Enhanced `prr` by introducing person and tone configurations, simplifying the command structure.
- Added initial commands like `init` and `generate`.

## How to test?
Ensure the `prr` can successfully read the last commits and include them in the PR description. Check that the README file now contains clear usage and requirements information. Verify that the person and tone configurations work as expected, and the command structure is simplified. Test the functionality of the new commands `init` and `generate`.
